docs/modules/deployment/pages/upgrade/diff.adoc

```diff
@@ -0,0 +1,28 @@
+
+[[run_diff]]
+= Identify Changed Configuration Files
```

Comment:
Personally, I like making the $OPENNMS_HOME/etc folder a git repo to make it easier to track changes between versions. Is that something we don't want to touch on?  the `config-diff.sh` script shows which files are changed, whereas using git will help show which content lines have changed as well.

Reply:
That's valid. Should we list the git solution as an alternate one? Is it something most users will be familiar with doing?